tsconfig.json

```diff
@@ -8,8 +8,6 @@
     "jsx": "react-native",
     "isolatedModules": true,
     "importsNotUsedAsValues": "error",
-    // strictFunctionTypes is implictly enabled by strict mode but
```

Comment:
Typescript is chill with having comments in this file, but a recent SK change now tries to read this as plain json, and throws up when we have comments.

I still want to stop disabling strictFunctionTypes eventually but not now.